src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -130,7 +130,14 @@ public void logNotificationFailure(
     }
 
     private Map<String, String> aggregateParams(final Claim claim, final ResponseData responseData) {
-        boolean isFreeMediationRequested = responseData.getFreeMediation().equals(ResponseData.FreeMediationOption.YES);
+        boolean isFreeMediationApplicable = responseData.getFreeMediation().isPresent();
+        boolean isFreeMediationRequested = false;
+
+        if (isFreeMediationApplicable) {
+            isFreeMediationRequested = responseData
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3655)